egs2/ksponspeech/asr1/RESULTS.md

```diff
@@ -0,0 +1,30 @@
+# RESULTS
+## Dataset
+- KsponSpeech: Korean Spontaneous Speech Corpus for Automatic Speech Recognition
+  - Database: https://aihub.or.kr/aidata/105
+  - Paper: https://www.mdpi.com/2076-3417/10/19/6936
+  - This corpus contains 969 h of general open-domain dialog utterances, spoken by 2000 native Korean speakers.
+
+## Environments
+- date: `Sun Aug  1 11:02:20 EDT 2021`
+- python version: `3.8.5 (default, Sep  4 2020, 07:30:14)  [GCC 7.3.0]`
+- espnet version: `espnet 0.10.0`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `449c0f84fd0d8a851b5be29c0f7e4717a5596b98`
+  - Commit date: `Fri Jul 30 11:47:21 2021 -0400`
+
```

Comment:
Can you upload a model to zenodo by following https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#132-espnet2-recipes ?
Then, you can also put the model link here.

Reply:
Uploaded model and added link.